gammapy/spectrum/spectrum_results.py

```diff
@@ -203,6 +203,20 @@ def from_json(data):
 
         return model
 
+    @staticmethod
+    def from_sherpa(covar):
+        """Read sherpa fit output
+
+        Parameters
+        ----------
+        covar : `~sherpa.fit.ErrorEstResults`
+            Fit results as return by :func:`~sherpa.astro.datastack.covar`
+        """
+        model = SpectralModelPowerLaw()
+        model.meta = Bunch(covar.__dict__)
```

Comment:
@cdeil: in the from_json method, the json file is more or less written as it is into one attribute ``meta`` of the SpectralModel. I can do the same with the sherpa fit output, but this will lead to problems sooner or later I guess. 
Do you have a proposal how to make this coherent (ASAP, not a final perfect solution)